src/sub-blocks/BackgroundCard/BackgroundCard.tsx

```diff
@@ -46,16 +51,30 @@ const BackgroundCard = (props: BackgroundCardProps) => {
                     style={{backgroundColor}}
                 />
                 <Content
+                    titleId={titleId}
                     title={title}
                     text={text}
                     additionalInfo={additionalInfo}
                     size="s"
                     theme={cardTheme}
-                    links={links}
-                    buttons={buttons}
+                    links={controlPosition === 'content' ? links : undefined}
+                    buttons={controlPosition === 'content' ? buttons : undefined}
                     colSizes={{all: 12, md: 12}}
                 />
             </CardBase.Content>
+            <CardBase.Footer className={b('footer')}>
```

Comment:
{ `controlPosition === 'footer' && <CardBase.Footer className={b('footer')}>...`

Reply:
That does not work that way =)
The `CardBase` component requires that children type will `ReactElement | ReactElement[]` I thought about separate child creation but it looks quite ugly.